x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/effected_policy_select/effected_policy_select.tsx

```diff
@@ -138,48 +160,54 @@ export const EffectedPolicySelect = memo<EffectedPolicySelectProps>(
 
     return (
       <EffectivePolicyFormContainer>
-        <EuiFormRow
-          fullWidth
-          label={
+        <EuiText size="xs">
+          <h3>
+            <FormattedMessage
+              id="xpack.securitySolution.trustedapps.policySelect.assignmentSectionTitle"
+              defaultMessage="Assignment"
+            />
+          </h3>
+        </EuiText>
+        <EuiSpacer size="xs" />
+        <EuiFlexGroup>
+          <EuiFlexItem grow={2}>
```

Comment:
This probably should be `grow={true}` instead of `2`. and the one below (that holds the button) should be `grow={false}

Reply:
![image](https://user-images.githubusercontent.com/56409205/134256966-c49a1700-4b8b-4d42-b12b-22ae50cef386.png)

keeping it as is bcuz making `grow={true}` messes up the per policy button when it is selected and has the check mark

Reply:
Intersting 🤔 
Sounds good. Thanks for trying it.